packages/mobile/src/components/premium-track-purchase-drawer/PurchaseSuccess.tsx

```diff
@@ -29,12 +32,22 @@ const useStyles = makeStyles(({ spacing, typography, palette }) => ({
     ...flexRowCentered(),
     alignSelf: 'center',
     gap: spacing(2)
+  },
+  viewTrack: {
```

Comment:
do we not have a button variant with no borders? like a link button or sthg?

Reply:
This is a good question to ask in the harmony channel. We're using a `PlainButton` on web, but I'm not sure if the corresponding version exists in mobile or if they are planning to make one.